doc/reference/index.md

```diff
@@ -55,6 +55,17 @@ Production server settings </reference/server_settings>
 /reference/provided_metrics
 ```
 
+## Fine-grained permissions
+
+If you are managing user access via {ref}`fine-grained-authorization`, you can check the available {ref}`permissions <permissions>` that can be assigned to groups.
```

Comment:
```suggestion
If you are managing user access via {ref}`fine-grained-authorization`, check which {ref}`permissions <permissions>` can be assigned to groups.
```

Reply:
"you can" is a bit misleading since it's totally possible and allowed to check the permissions even if you don't use them. ;)
(Yes, nitpicking here :smile: )